nvflare/fuel/f3/streaming/byte_receiver.py

```diff
@@ -140,6 +146,29 @@ def register_callback(self, channel: str, topic: str, stream_cb: Callable, *args
 
         self.registry.set(channel, topic, Callback(stream_cb, args, kwargs))
 
+    def stop_task(self, task: RxTask, error: StreamError = None, notify=True):
+
+        with self.map_lock:
+            self.rx_task_map.pop(task.sid, None)
```

Comment:
@chesterxgchen these parts for memory leak.

Previous code did NOT pop these so the reference of task was kept.
Thus Task data was kept, thus memory not getting garbage collected.